lxd/patches.go

```diff
@@ -37,7 +38,7 @@ var patches = []patch{
 	{name: "storage_api", run: patchStorageApi},
 	{name: "storage_api_v1", run: patchStorageApiV1},
 	{name: "storage_api_dir_cleanup", run: patchStorageApiDirCleanup},
-	{name: "storage_api_lvm_keys", run: patchStorageApiLvmKeys},
+	{name: "storage_api_zfs_lvm_keys", run: patchStorageApiZfsLvmKeys},
```

Comment:
Please make this a separate patch and keep storage_api_lvm_keys as it is. Otherwise the few folks who use git master will end up with a patch in their database that doesn't have any matching code (not going to break anything, but annoying for debugging).